src/components/Layout/PageTransition.tsx

```diff
@@ -6,6 +6,7 @@ export const PageTransition = (props: HTMLMotionProps<'div'>) => {
   const [isLargerThanMd] = useMediaQuery(`(min-width: ${breakpoints['md']})`)
   return (
     <motion.div
+      // The minHeight calculation is not correct if Demo Wallet banner is displaying
       initial={{ y: -16, opacity: 0, minHeight: isLargerThanMd ? 'calc(100vh - 4.5rem)' : 'auto' }}
```

Comment:
I took a look and:

- On really small screens, you'll be pretty forced to get a scroll because of the place the content needs, it will be pretty impossible to avoid except if we change some design/paddings and whatever, which we probably don't want only for the demo wallet feature
- But you can still avoid that scroll on bigger screens (I tested with 1920px) if you use the `useWallet` hook there and calculate it properly in case you are on a `DemoWallet` context

Reply:
Hmm, I tried with `useWallet` but kept getting stuck either with an error about `cannot access state of null`, or it would not actually update once the state had loaded. Did you not come across either of these issues?

I had assumed that `initial` property is calculated before the state is "ready", and since it's only the initial state, is immutable.

However, after taking another look, I'm wondering if we even need the `calc(100vh - 4.5rem)` on larger screens. I'm still trying to figure out why it was added in the first place. I don't know if something has changed from when this was originally implemented, but it seems to still animate nicely using just 
`initial={{ y: -16, opacity: 0, minHeight: 'auto' }}`